mycroft/res/ui/SeekControl.qml

```diff
@@ -0,0 +1,272 @@
+import QtMultimedia 5.12
+import QtQuick.Layouts 1.4
+import QtQuick 2.9
+import QtQuick.Controls 2.12 as Controls
+import org.kde.kirigami 2.10 as Kirigami
+import QtQuick.Templates 2.2 as Templates
+import QtGraphicalEffects 1.0
+
+import Mycroft 1.0 as Mycroft
+
+Item {
+    id: seekControl
+    property bool opened: false
+    property int duration: 0
+    property int playPosition: 0
+    property int seekPosition: 0
+    property bool enabled: true
+    property bool seeking: false
+    property var videoControl
+    property string title
+
+    clip: true
+    implicitWidth: parent.width
+    implicitHeight: mainLayout.implicitHeight + Kirigami.Units.largeSpacing * 2
+    opacity: opened
+
+    onOpenedChanged: {
+        if (opened) {
+            hideTimer.restart();
+        }
+    }
+    
+    onFocusChanged: {
+        if(focus) {
+            backButton.forceActiveFocus()
+        }
+    }
+
+    Timer {
+        id: hideTimer
+        interval: 5000
+        onTriggered: { 
+            seekControl.opened = false;
+            videoRoot.forceActiveFocus();
+        }
+    }
+    
+    Rectangle {
+        width: parent.width
+        height: parent.height
+        color: Qt.rgba(0, 0, 0, 0.8)
+        //color: "white"
+        y: opened ? 0 : parent.height
+
+        ColumnLayout {
+            id: mainLayout
+            
+            anchors {
+                fill: parent
+                margins: Kirigami.Units.largeSpacing
+            }
+            
+            RowLayout {
+                id: mainLayout2
+                Layout.fillHeight: true
+                Controls.RoundButton {
+                    id: backButton
+                    Layout.preferredWidth: parent.width > 600 ? Kirigami.Units.iconSizes.large : Kirigami.Units.iconSizes.medium
+                    Layout.preferredHeight: Layout.preferredWidth
+                    highlighted: focus ? 1 : 0
+                    z: 1000
+                    
+                    background: Rectangle {
+                        radius: 200
+                        color: Qt.rgba(0, 0, 0, 0.9)
+                        border.width: 1
+                        border.color: "white"
+                    }
+                    
+                    contentItem: Item {
+                        Image {
+                            width: parent.width > 600 ? Kirigami.Units.iconSizes.medium : Kirigami.Units.iconSizes.small
+                            height: width
+                            anchors.centerIn: parent
+                            source: "images/back.png"
```

Comment:
this was taken from @AIIX youtube skill

Reply:
it can use an svg but are there any svg's available for this, not sure if we can just pick something of the internet cause of potential license issues or is there any creatives common license compatibility that too should be good